diff --git a/config.cfg b/config.cfg
index 71ebab0..ff6e566 100644
--- a/config.cfg
+++ b/config.cfg
@@ -245,3 +245,15 @@ cpu_model=      1,8,17,24
 lines=          1536
 instr_groups=   RAM_L,L3_L,L2_L,L1_LS,REG
 proportion=     8,33,81,79,100
+
+[Rome denormals test]
+arch=zen2
+model=epyc
+threads=2
+isa=fma
+buffer_sizes=0,0,0,0
+cpu_family=23
+cpu_model=49
+lines=1536
+instr_groups=REG
+proportion=1
